pom.xml

```diff
@@ -18,6 +18,7 @@
     <java.level>8</java.level>
     <hpi.pluginChagelogUrl>https://github.com/jenkinsci/cloudbees-jenkins-advisor-plugin/releases</hpi.pluginChagelogUrl>
     <hpi.pluginLogoUrl>https://raw.githubusercontent.com/jenkinsci/cloudbees-jenkins-advisor-plugin/master/src/main/webapp/icons/advisor.svg</hpi.pluginLogoUrl>
+    <jcasc.version>1.30</jcasc.version>
```

Comment:
```suggestion
    <jcasc.version>1.32</jcasc.version>
```

Reply:
Could you use the bom instead? We shouldn’t need to have this conversation, bom currently has 1.30 and will be 1.32 on the next release

Reply:
@timja  if it is in the BOM let's use it. I already updated the project to use the BOM
https://github.com/jenkinsci/cloudbees-jenkins-advisor-plugin/blob/master/pom.xml#L76
WDYT @fcojfernandez ?

Reply:
I will commit it